tests/testthat/test-parse_exclusions.R

```diff
@@ -150,7 +150,7 @@ test_that("it throws an error if start and end are unpaired", {
     a
     test
   "))
-  expect_error(lintr:::parse_exclusions(t1), "but only")
+  expect_error(lintr:::parse_exclusions(t1), "and")
```

Comment:
this looks too weak, "and" could appear in any number of error messages, care to revise?